src/vcbuild/dmd_backend.vcxproj

```diff
@@ -1,258 +1,257 @@
-﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I suspect you have converted newlines in the VS projects.

Reply:
I don't think it's mandatory, but the next one who saves it from within VS will change the whole file again.